Infrastructure.DataAccess/Migrations/202207281043469_Updated_AdviceUserRelation.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Infrastructure.DataAccess.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class Updated_AdviceUserRelation : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.AdviceUserRelations", "RoleId", c => c.Int());
```

Comment:
not just an integer. It must be a foreign key

Reply:
an optional one

Reply:
So there should actually be one id per type of role so that we get the right relationships.. on the DTO level we can cheat a bit and have only one and a type when we write